### ISSUE TITLE
Qemu error after restarting Debian x86 VM

### ISSUE THREAD

**Issue by LostInCompilation** [utmapp/UTM#6902]
**Describe the issue**  
Today I received the error down below after restaring a Debian x86 VM with no GUI installed. It happened only once, cant reproduce it yet.

<img width="788" alt="Screenshot 2024-12-17 at 08 14 13" src="https://github.com/user-attachments/assets/4eca3a8a-1aec-4ca0-b4d6-419e75a5d546" />

**Configuration**  
* UTM Version: 4.6.3
* macOS Version: 15.2
* Mac Chip: M1 Pro

**Crash log**  
[QEMULauncher-2024-12-17-081345.ips.zip](https://github.com/user-attachments/files/18161860/QEMULauncher-2024-12-17-081345.ips.zip)

**Upload VM**  
[config.plist.zip](https://github.com/user-attachments/files/18162179/config.plist.zip)

**Comment by osy** [comment]
Yeah, I think this is some latent error since it's not really possible to cleanly exit QEMU in some cases.
